app/src/main/res/navigation/nav_graph.xml

```diff
@@ -179,4 +182,17 @@
             android:id="@+id/action_homeFragment_to_learningDatasetSelectionFragment"
             app:destination="@id/learningDatasetSelectionFragment" />
     </fragment>
+    <fragment
+        android:id="@+id/evaluationResultFragment"
+        android:name="com.github.HumanLearning2021.HumanLearningApp.view.learning.EvaluationResultFragment"
+        android:label="EvaluationResultFragment">
```

Comment:
This could be a bit more user friendly I think and  use a string ressource (I propore something like "Evaluation result")

Reply:
Yep, was the default but I've changed it now.